harmonica/forward/prism_layer.py

```diff
@@ -469,3 +487,44 @@ def to_pyvista(self):
                 for data_var in self._obj.data_vars
             }
         return prism_to_pyvista(prisms, properties=properties)
+
+
+def _discard_thin_prisms(
+    prisms,
+    density,
+    thickness_threshold,
+):
+    """
+    Discard prisms with a thickness below a threshold
+
+    Parameters
+    ----------
+    prisms : 2d-array
+        Array containing the boundaries of the prisms in the following order:
+        ``w``, ``e``, ``s``, ``n``, ``bottom``, ``top``.
+        The array must have the following shape: (``n_prisms``, 6), where
+        ``n_prisms`` is the total number of prisms.
+    density : 1d-array
+        Array containing the density of each prism in kg/m^3. Must have the
+        same size as the number of prisms.
+    thickness_threshold : int or float
+        Minimum prism thickness needed to be included in the forward
+        gravity calculation. Prisms thinner that this threshold will be
+        ignored. Default to 0.
```

Comment:
Update this description as well:
```suggestion
    thickness_threshold : float
        Prisms thinner than this threshold will be discarded.
```